<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableSelectParaDialog</class>
 <widget class="QDialog" name="tableSelectParaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>533</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Table Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tableNamele"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Runs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="runssb"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="addInputButton">
            <property name="text">
             <string>Add Input</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeInputButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="inputList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="addOutputButton">
            <property name="text">
             <string>Add Output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="COPButton">
            <property name="text">
             <string>Add System COP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CapacityButton">
            <property name="text">
             <string>Add System Capacity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeOutputButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="outputList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>OKButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>tableNamele</tabstop>
  <tabstop>runssb</tabstop>
  <tabstop>inputList</tabstop>
  <tabstop>addInputButton</tabstop>
  <tabstop>removeInputButton</tabstop>
  <tabstop>outputList</tabstop>
  <tabstop>addOutputButton</tabstop>
  <tabstop>COPButton</tabstop>
  <tabstop>CapacityButton</tabstop>
  <tabstop>removeOutputButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
